src/pudl_archiver/archivers/epacems.py

```diff
@@ -65,27 +104,33 @@ async def get_year_quarter_resource(
                 for expected format of dictionary.
             request_count: the number of the request.
         """
-        url = self.base_url + file["s3Path"]
-        year = file["metadata"]["year"]
-        quarter = file["metadata"]["quarter"]
-
-        # Useful to debug at download time-outs.
-        logger.info(f"Downloading {year} Q{quarter} EPACEMS data from {url}.")
-
-        # Create zipfile to store year/quarter combinations of files
-        filename = f"epacems-{year}-{quarter}.csv"
-        archive_path = self.download_directory / f"epacems-{year}-{quarter}.zip"
-        # TODO: When we are ready to make a new archive, we want to transition to
-        # using this file formart so its more in line with the partition.
-        # filename = f"epacems-{year}q{quarter}.csv"
-        # archive_path = self.download_directory / f"epacems-{year}q{quarter}.zip"
-        # Override the default asyncio timeout to 14 minutes, just under the API limit.
-        await self.download_and_zip_file(
-            url=url, filename=filename, archive_path=archive_path, timeout=60 * 14
-        )
-        logger.info(  # Verbose but helpful to track progress
-            f"File no. {request_count}: Downloaded {year} Q{quarter} EPA CEMS hourly emissions data."
-        )
+        archive_path = self.download_directory / f"epacems-{year}.zip"
+        for file in files:
+            url = self.base_url + file.s3_path
+            quarter = file.metadata.quarter
+
+            # Useful to debug at download time-outs.
+            logger.info(f"Downloading {year} Q{quarter} EPACEMS data from {url}.")
+
+            # Create zipfile to store year/quarter combinations of files
+            filename = f"epacems-{year}q{quarter}.csv"
+            file_path = self.download_directory / filename
+
+            await self.download_file(url=url, file=file_path, timeout=60 * 14)
+
+            with zipfile.ZipFile(
```

Comment:
I didn't quite want to mess with how `download_file_to_zip` worked for *all* archivers yet, so I inlined this. But I think probably a more useful thing to do is have a couple useful utility functions like "download file (raw)," "download file (zip it)," and "download file (add to an existing archive)" that aren't coupled to the class hierarchy. Out of scope for this PR, I think.